### ISSUE TITLE
`Manage Nuget Packages for Solution` not showing packages installed per project 

### ISSUE THREAD

**Issue by WilliamBerryiii** [NuGet/Home#3236]
I have a solution with several projects and when viewing the Nuget packages at the solution level, the Versions/Project pane for a package does not show the boxes checked, though the package is installed.

![image](https://cloud.githubusercontent.com/assets/4399970/17201798/c3d6473c-5446-11e6-8031-cebe0215418a.png)

The solution is composed of the following projects:
- AspNetCore x-plat MVC application targeting netcoreapp1.0 & net461
- Windows Service Wrapper - targeting net461
- Unit test project (using F# 4.0 & targeting Net461)
- Core lib targeting netstandard1.6 & net461

Currently running: 

Microsoft Visual Studio Enterprise 2015
Version 14.0.25424.00 Update 3
Microsoft .NET Framework
Version 4.6.01055

Installed Version: Enterprise

Architecture and Modeling Tools   00322-80000-00000-AA809
Microsoft Architecture and Modeling Tools

UML® and Unified Modeling Language™ are trademarks or registered trademarks of the Object Management Group, Inc. in the United States and other countries.

Microsoft Visual Studio Tools for Applications 2015   00322-80000-00000-AA809
Microsoft Visual Studio Tools for Applications 2015

Visual Basic 2015   00322-80000-00000-AA809
Microsoft Visual Basic 2015

Visual C# 2015   00322-80000-00000-AA809
Microsoft Visual C# 2015

Visual C++ 2015   00322-80000-00000-AA809
Microsoft Visual C++ 2015

Visual F# 2015   00322-80000-00000-AA809
Microsoft Visual F# 2015

Application Insights Tools for Visual Studio Package   7.1.00719.1
Application Insights Tools for Visual Studio

ASP.NET and Web Tools 2015.1   14.1.20624.0
ASP.NET and Web Tools 2015.1

ASP.NET Web Frameworks and Tools 2012.2   4.1.41102.0
For additional information, visit http://go.microsoft.com/fwlink/?LinkID=309563

ASP.NET Web Frameworks and Tools 2013   5.2.40314.0
For additional information, visit http://www.asp.net/

Azure App Service Tools v2.9   14.0.20316.0
Azure App Service Tools v2.9

Azure Data Lake Node   1.0
This package contains the Data Lake integration nodes for Server Explorer.

Azure Data Lake Tools for Visual Studio   2.0.6000.0
Microsoft Azure Data Lake Tools for Visual Studio

Code alignment   12.1
Code alignment helps you present your code beautifully, enhancing clarity and readability.

Common Azure Tools   1.8
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

GhostDoc   5.2.16200.0
GhostDoc automatically generates XML documentation comments.

GitHub.VisualStudio   1.0
A Visual Studio Extension that brings the GitHub Flow into Visual Studio.

JavaScript Language Service   2.0
JavaScript Language Service

JavaScript Project System   2.0
JavaScript Project System

JetBrains ReSharper Ultimate 10.0.2   Build 104.0.20151218.120627
JetBrains ReSharper Ultimate package for Microsoft Visual Studio. For more information about ReSharper Ultimate, visit http://www.jetbrains.com/resharper. Copyright © 2016 JetBrains, Inc.

Microsoft .NET Core Tools (Preview 2)   14.1.20624.0
Microsoft .NET Core Tools (Preview 2)

Microsoft Azure HDInsight HQL Service   2.0.6000.0
Language service for Hive query

Microsoft Azure HDInsight Tools for Visual Studio   2.0.6000.0
An integrated development environment for HDInsight application development.

Microsoft Azure Mobile Services Tools   1.4
Microsoft Azure Mobile Services Tools

Microsoft Azure Tools   2.8
Microsoft Azure Tools for Microsoft Visual Studio 2015 - v2.8.40211.2

Microsoft Azure Tools   2.9
Microsoft Azure Tools for Microsoft Visual Studio 2015 - v2.9.40323.3

NuGet Package Manager   3.5.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit http://docs.nuget.org/.

Paket.VisualStudio   1.0
Manage your Paket dependencies from Visual Studio!

PreEmptive Analytics Visualizer   1.2
Microsoft Visual Studio extension to visualize aggregated summaries from the PreEmptive Analytics product.

Python Tools for Visual Studio   2.2.40623.00
Python Tools for Visual Studio provides IntelliSense, projects, templates, Interactive windows, and other support for Python developers.

Python Tools for Visual Studio - Django Integration   2.2.40623.00
Provides templates and integration for the Django web framework.

Python Tools for Visual Studio - Profiling Support   2.2.40623.00
Profiling support for Python projects.

SQL Server Analysis Services   13.0.1605.88
Microsoft SQL Server Analysis Services Designer 
Version 13.0.1605.88

SQL Server Data Tools   14.0.60629.0
Microsoft SQL Server Data Tools

SQL Server Integration Services  
Microsoft SQL Server Integration Services Designer
Version 13.0.1601.5

SQL Server Reporting Services   13.0.1605.88
Microsoft SQL Server Reporting Services Designers 
Version 13.0.1605.88

ToolWindowHostedEditor   1.0
Hosting json editor into a tool window

TypeScript   1.8.35.0
TypeScript tools for Visual Studio

Visual F# Power Tools   2.5.0
A collection of additional commands for F# in Visual Studio

Let me know if there is other information I can supply about the system configuration, etc. to help isolate the UI issue.

**Comment by rrelyea** [comment]
@WilliamBerryiii - can you please try a recent nightly build of our VSIX: https://www.myget.org/F/nuget-vsix/vsix/NuGet.0d421874-a3b2-4f67-b53a-ecfce878063b-3.5.0.1666.vsix 
 And let us know if it still has that problem.

**Comment by WilliamBerryiii** [comment]
@rrelyea sorry for the delay in testing this ... things are looking interesting:

I ran the installer for the vsix you provided above and restarted Visual Studio.  I then promptly ran `nuget help` in the package manager console to get the version information which yielded: `NuGet Version: 3.4.4.1321`.  Realizing that nuget seemingly failed to update I ran `nuget update -self` which showed the outdated RTM build of 1321 and that the `-final` was available. - the updater ran successfully.

![image](https://cloud.githubusercontent.com/assets/4399970/17301094/2775c0ca-57ca-11e6-8ffb-4179509b4096.png)

Of course re-starting Visual Studio and re-issuing the `nuget help` command yielded: 

![image](https://cloud.githubusercontent.com/assets/4399970/17301123/526546fc-57ca-11e6-9d53-bc21f259394a.png)

Interestingly ... the Visual Studio About window is reporting 3.5.0 installed: 

![image](https://cloud.githubusercontent.com/assets/4399970/17301178/8f24e1f6-57ca-11e6-971d-37031c45b63d.png)

And unsurprisingly ... the issue persists: 

![image](https://cloud.githubusercontent.com/assets/4399970/17301242/d518ce48-57ca-11e6-8157-7e8e6948c164.png)

Thoughts on how I can nuke this old _stuck_ version of nuget?  Re-install Visual Studio?
